openshift/secrets-template.yaml

```diff
@@ -237,3 +289,15 @@ parameters:
   name: SNYK_ISS
   value: "bm90LXNldA=="  # not-set
 
+- description : snyk encryption key for token
+  displayName: snyk encryption key
+  required: false
+  name: ENCRYPTION_KEY_FOR_SNYK_TOKEN
+  value: "bm90LXNldA=="  # not-set
+
+- description : deployment prefix
+  displayName: Ddeployment prefix
+  required: false
+  name: DEPLOYMENT_PREFIX
+  value: ""  # not-set
```

Comment:
'# net-set' at the end is not a default value for this parameter, it can be remove.

Reply:
Thats just a comment saying that the value is notset.

Reply:
In yaml file, it is the actual value that is set to the parameter.  Base64 for "not-set" is "bm90LXN1dA=="
So, it will confuse reader when we keep '# not-set' for value with empty data.

Reply:
Done